### PR TITLE
Improved explanation of embedding sub-resources

### LINKED ISSUE #67: Add example for embedding resources
The current [section ](http://zalando.github.io/restful-api-guidelines/hyper-media/Hypermedia.html#should-allow-optional-embedding-of-subresources) doesn't contain a useful example. See [HAL](http://stateless.co/hal_specification.html).

### PR DESCRIPTION
**Pull request by fmueller**
This PR does the following:
- moved embedding to performance chapter
- removed custom media type for embedding
- added link to json-field (to make embedding symmetric to filtering)
- added example for embedding

Fixes #67

### CODE REVIEW

File: performance/Performance.md [hunk 1]
```diff
@@ -98,5 +98,34 @@ Its possible contents:
...
+
+See [*Conventional Query Strings*](../naming/Naming.md#could-use-conventional-query-strings) for naming. Please use [json-fields](https://github.com/zalando/json-fields) library, already mentioned above for filtering, when it comes to an embedding query syntax.
+
+Embedding a sub-resource can possibly look like this where an order resource has its order items as sub-resource (/order/{orderId}/items):
+
+```http
+GET /order/123?embed=(items) HTTP/1.1
+
+{
+  "orderId": 123,
```
Comment (whiskeysierra) [inline performance/Performance.md]: This is violating our own naming rules. Should be just `id`.

File: performance/Performance.md [hunk 2]
```diff
@@ -98,5 +98,34 @@ Its possible contents:
...
+  "items": [
```
Comment (whiskeysierra) [inline performance/Performance.md]: If we use HAL this should go under `_embedded`.

Comment (whiskeysierra) [inline performance/Performance.md]:
```json
{
  "id": "123",
  "_embedded": {
    "items": [
      {
        "position": 1,
        "sku": "1234-ABCD-7890",
        "price": {
          "amount": 71.99,
          "currency": "EUR"
        }
      }
    ]
  }
}
```

(Please note that id is also a string, not a number.)

**Comment by fmueller** [comment]
@whiskeysierra Thanks for the feedback. I addressed your comments.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by fmueller** [comment]
@ePaul @codekultur @tkrop please have a look at this :)

**Comment by ePaul** [comment]
:+1:

**Pull request merged**